docs/_tutorials/reference.md

```diff
@@ -98,7 +98,7 @@ Bolt includes a collection of initialization options to customize apps. There ar
 | `installerOptions` | Optional object that can be used to customize [the default OAuth support](/bolt-js/concepts#authenticating-oauth). Read more in the OAuth documentation. |
 
 ### App options
-App options are passed into the `App` constructor.
+App options are passed into the `App` constructor. The `App` constructor also accepts [`HttpReceiver`](#receiver-options) options which will be used to initialise a [Receiever](/bolt-js/concepts#receiver) when the `receiver` argument is `undefined` and are ignored otherwise.
```

Comment:
Mentioning `HttpReceiver` options is correct but it can be confusing for the people new to bolt-js. We can simply say the receiver options above are also available in the argument list.

```suggestion
App options are passed into the `App` constructor. Also, when the `receiver` argument is `undefined`, the `App` constructor accepts [the above receiver options](#receiver-options) to initialize an appropriate `Receiver` for you. `
```

I didn't mention `HTTPReceiver` because it can be `SocketModeReceiver` if `socketMode: true`.

Reply:
Ah, right. I had my blinders on when making this change I didn't realise it was the generic `Receiver` options and not specific to `HttpReceiver`. How would you feel about mentioning `socketMode` and calling out the specific Receivers that could be constructed?

Reply:
Yeah, that sounds good to me 👍